README.md

```diff
@@ -1,11 +1,26 @@
 # Kubernetes on CoreOS
 
-Follow the Kubernetes guides on the CoreOS website:
+This repo contains tooling and documention around deploying Kubernetes using CoreOS.
+Initial setup of a Kubernetes cluster is covered, but ongoing maintenance and updates of the cluster is not addressed.
```

Comment:
Nit: Don't think it's necessary to call out that not covering updates/maintenance.